pkg/agent/util/net_windows.go

```diff
@@ -283,3 +286,9 @@ func enableHNSOnOVS(hnsNet *hcsshim.HNSNetwork) error {
 	}
 	return err
 }
+
+func GetLocalBroadcastIP(ipNet *net.IPNet) net.IP {
```

Comment:
This is not really a Windows specific func, but if you have no other common file to put it, we might put it in net_windows.go for now.

Reply:
Could you add some comments to this func? Not easy to figure out its usage from code.